src/script/verification.js

```diff
@@ -0,0 +1,163 @@
+import {
+    validateEmail,
+    validatePassword,
+    email,
+    password,
+    emailError,
+    passwordError,
+} from "./sign.js";
+import { createError, clearError } from "./error/errorRelated.js";
+
+export const confirmPassword = document.querySelector("#sign_confirm_password");
+const confirmPasswordError = document.querySelector("#error_confirm_password");
+const emailRegex = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-za-z0-9\-]+/;
+const passwordRegex = /^(?=.*[A-Za-z])(?=.*\d).{8,}$/;
+const userInfo = {
+    email: "test@codeit.com",
+    password: "codeit101",
+};
+
+// 이메일 유효성 검사
+export function isValidEmail(email) {
+    return emailRegex.test(email);
+}
+
+// 비밀번호 유효성 검사
+export function isValidPassword(password) {
+    return passwordRegex.test(password);
+}
+
+// 이메일 중복 검사
+export function checkEmail(email) {
+    const emailData = {
+        email: email,
+    };
```

Comment:
프로퍼티명과 변수명이 동일한 경우 아래처럼도 작성 가능합니다.

```suggestion
    const emailData = {
        email,
    };
```

Reply:
https://javascript.info/object#property-value-shorthand